### PR TITLE
Updated statinamic 0.7 -> 0.8

### PR DESCRIPTION
**Pull request by marcustisater**
@thangngoc89 - can you double check for me? Thanks!

### CODE REVIEW

**Comment by marcustisater** [comment]
404 pages aren't working. I did something wrong... :thought_balloon:

**Comment by thangngoc89** [comment]
> 404 pages aren't working. I did something wrong... 

You need to have a markdown file that reference this layout https://github.com/thangngoc89/blog-posts/blob/master/404.md

File: web_modules/layouts/Page/index.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default class Page extends Component {
       { name: "twitter:title", content: head.title },
       { name: "twitter:creator", content: `@${ pkg.twitter }` },
       // { name: "twitter:description", content: pageDescription(body) },
```
Comment (thangngoc89) [inline web_modules/layouts/Page/index.js]: Description will be automatically generated. You should use it :)

**Comment by thangngoc89** [comment]
lgtm

**Comment by marcustisater** [comment]
Thanks @thangngoc89

**Comment by thangngoc89** [comment]
I forgot. You should add `BABEL_DISABLE_CACHE=1` to start script too. It will be default setting in 0.9.0. [Why ?](https://github.com/istarkov/babel-plugin-webpack-loaders/issues/36)

**Comment by marcustisater** [comment]
Yeah, I forgot to add that **BUT** it slows down the "build server".

**Pull request merged**